pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -277,7 +277,7 @@
     "foscam" = ps: with ps; [ ]; # missing inputs: libpyfoscam
     "foursquare" = ps: with ps; [ aiohttp-cors ];
     "free_mobile" = ps: with ps; [ ]; # missing inputs: freesms
-    "freebox" = ps: with ps; [ aiohttp-cors netdisco zeroconf ]; # missing inputs: freebox-api
+    "freebox" = ps: with ps; [ aiohttp-cors freebox-api netdisco zeroconf ];
```

Comment:
`component-packages` should only add `pyturbojpeg`. Easier to revert if needed. The other two dependencies will be added with the Home Assistant update to 2021.4.X.